shared/api/storage_pool_volume.go

```diff
@@ -10,6 +10,16 @@ type StorageVolumesPost struct {
 	Type string `json:"type" yaml:"type"`
 }
 
+// StorageVolumePost represents the fields required to rename a LXD storage pool volume
+//
+// API extension: storage_api_volume_rename
+type StorageVolumePost struct {
+	StorageVolumePut `yaml:",inline"`
```

Comment:
This is wrong, the rename struct should only contain the new Name (and once we add support for moving pools, the new pool).